doc/chapters/filesystem/index.rst

```diff
@@ -0,0 +1,123 @@
+
+.. _filesystems:
+
+Filesystems
+***********
+
+The filesystem behavior and layout is documented in this chapter. The document
+discusses how the container filesystem is created, where the different mount 
+points are pulled in from and the mechanisms that changes the behavior of the 
+filesystem. 
+
+
+Standard filesystem
+===================
+
+Each container has a rootfs that is bind mounted as the containers root file
+system of said container. The process of creating the filesystem and bind
```

Comment:
This is still wrong. The rootfs isn't bind mounted - the container is chrooted into it, no? We bind mount things INTO the rootfs of the container.